### PR TITLE
ENH: No highpass

### PR DESCRIPTION
**Pull request by larsoner**
Overleaf updated:

https://www.overleaf.com/6751905gspdys

My branch website updated:

https://larsoner.github.io/mne-biomag-group-demo/auto_examples/index.html#

### CODE REVIEW

File: scripts/processing/06-make_epochs.py [hunk 1]
```diff
@@ -112,15 +108,35 @@ def run_epochs(subject_id, tsss=False):
...
+    print('    Found %d ECG indices' % (len(ecg_inds),))
+    ica.exclude.extend(ecg_inds[:n_max_ecg])
+
+    n_max_eog = 3  # use max 3 components
+    eog_epochs = create_eog_epochs(raw, tmin=-.1, tmax=.1)
+    eog_epochs.apply_baseline((None, None))
+    eog_epochs.decimate(11, verbose='error')
+    eog_inds, scores_eog = ica.find_bads_eog(eog_epochs)
+    print('    Found %d EOG indices' % (len(eog_inds),))
+    ica.exclude.extend(eog_inds[:n_max_eog])
```
Comment (jasmainak) [inline scripts/processing/06-make_epochs.py]: why did you do this? Now, we have eyeblinks in the signals

Comment (larsoner) [reply scripts/processing/06-make_epochs.py]: Shouldn't this remove any residual blinks?

Comment (jasmainak) [reply scripts/processing/06-make_epochs.py]: But now you don't have EOG channel in autoreject and also the ICA components are not selected properly if you look at the website

Comment (larsoner) [reply scripts/processing/06-make_epochs.py]: Oh you're taking about the lines below, not this one.

In practice it didn't change much IIRC to include EOG in the list (as large EOG creates large EEG, too) but I'll put it back to be safe.

As for component selection I'm not sure how to improve it, ideas welcome. But we should do something as subjects do tend to blink after 800 ms.

**Comment by larsoner** [comment]
Okay latest changes pushed to GitHub alongside Overleaf and my website branch

**Comment by agramfort** [comment]
thanks @larsoner

**Pull request merged**